plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergPlugin.java

```diff
@@ -29,6 +30,8 @@
 
 public class TestIcebergPlugin
 {
+    private static final String HIVE_VIEWS_ENABLED = "hive.hive-views.enabled";
```

Comment:
The constant has value only if used in `@Config` annotation value of the `io.trino.plugin.hive.HiveConfig#setTranslateHiveViews` method.